### DISCUSSION TITLE
Adjust window width and level with Transforms

### DISCUSSION THREAD

**Discussion by pauljhin** [Project-MONAI/tutorials#1036] [Q&A] [answered]
Hello,

I would like to adjust the brightness and contrast of my CT images to the bone standard (W: 2000 L: 400). I tried using the ``` ScaleIntensityRange ``` but it doesn't seem to be working with the values. Is there any function that I can do this?

Is there also some function that just normalizes the values between 0 and 1?

Thanks in advance

**Comment by KumoLiu** [comment, marked as answer]
Hi @pauljhin, you can use `ScaleIntensity`. It can scale the intensity of input image to the given value range (minv, maxv).
https://github.com/Project-MONAI/MONAI/blob/89684d677f6c93c6fbd023a614c07485c5223557/monai/transforms/intensity/array.py#L408
Hope it can help you, thanks!

**Reply by pauljhin** [reply]
Hi @KumoLiu ,

Thank's for your reply! The part of scale (or normalize) the values from 0 to 1 worked. 

But I would like to know how to change brightness and contrast (window and level) like [this article](https://kevalnagda.github.io/ct-windowing). This is not worked.

**Reply by KumoLiu** [reply]
Hi @pauljhin, if you want to change the window level and window width. You can do this:

https://github.com/Project-MONAI/tutorials/blob/f43e771e7cf35427715baa16245e8a12dffe423a/deployment/Triton/models/monai_covid/1/model.py#L100

`a_min` and `a_max` are the upper and lower bounds of window width.
Thanks!

**Reply by pauljhin** [reply]
Thanks!

**Reply by hreso110100** [reply]
@KumoLiu so for example if for abdomen W is 350 and L is 40 then what are values of a_min and a_max ? Thanks

**Reply by KumoLiu** [reply]
Hi @hreso110100, you can calculate it by using:
- max - min = W
- (max + min) / 2 = 40

Thanks!
